Source/WebKit/WebProcess/Extensions/Interfaces/WebExtensionAPIScripting.idl

```diff
@@ -30,8 +30,8 @@
 ] interface WebExtensionAPIScripting {
 
     [RaisesException] void executeScript([NSDictionary] any details, [Optional, CallbackHandler] function callback);
-    [RaisesException] void insertCSS([NSDictionary] any details, [Optional, CallbackHandler] function callback);
-    [RaisesException] void removeCSS([NSDictionary] any details, [Optional, CallbackHandler] function callback);
+    [RaisesException, NeedsPage] void insertCSS([NSDictionary] any details, [Optional, CallbackHandler] function callback);
+    [RaisesException, NeedsPage] void removeCSS([NSDictionary] any details, [Optional, CallbackHandler] function callback);
```

Comment:
Weird that `executeScript` does not need this. Why do these allow implicit tab and executeScript does not? Might be worth mention in the WECG.

Reply:
In hindsight, it doesn't need it and this is wrong. I passed in the page here so that I could get the user content controller, but I should be getting the controllers from allUserContentControllers() from the extensions controller